src/main/java/se/sundsvall/messaging/api/model/request/DigitalInvoiceRequest.java

```diff
@@ -20,105 +23,66 @@
 import se.sundsvall.messaging.model.InvoiceType;
 import se.sundsvall.messaging.model.ReferenceType;
 
-import java.time.LocalDate;
-import java.util.List;
-
-import static io.swagger.v3.oas.annotations.media.Schema.RequiredMode.REQUIRED;
-import static org.springframework.http.MediaType.APPLICATION_PDF_VALUE;
-
 @With
 @Builder(setterPrefix = "with")
 public record DigitalInvoiceRequest(
 
-	@Valid
-	@NotNull
-	@Schema(description = "Party", requiredMode = REQUIRED)
-	Party party,
+	@Valid @NotNull @Schema(description = "Party", requiredMode = REQUIRED) Party party,
```

Comment:
Why inline the annotations with the fields?

Reply:
Done by the IDE when saving the file. It doesn't bother me though.